.github/workflows/test-fakeredis.yml

```diff
@@ -73,7 +73,9 @@ jobs:
       - name: Run tests
         working-directory: tests/fakeredis
         run: |
-          poetry run pytest test/ --ignore test/test_hypothesis.py  --ignore test/test_json/ \
+          poetry run pytest test/ \
+            --ignore test/test_hypothesis.py  \
+            --ignore test/test_json/ \
```

Comment:
Thanks, can you please add ignore test_bitmap_commands.py and 
test_geo_commands.py (will save me a PR) ? 
these are not very urgent to fix.

Reply:
`test_geo_commands.py` was not in the ignore list - is there another point where it is marked to be ignored?

Added `test_bitmap_commands.py`

Reply:
I meant to add them here as `--ignore ...` because they are failing on Dragonfly. I think you removed test_bitmap_commands instead.

Reply:
Done :)
I thought you meant add them to run, not add them to the ignore list.